### PR TITLE
[netatmo] Switching siren channel of the Presence to Read-Only

### PR DESCRIPTION
**Pull request by clinique**
Resolves issue #16707 
This channel is not supported by Netatmo API in write mode.

### CODE REVIEW

File: bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/PresenceCapability.java [hunk 1]
```diff
@@ -57,9 +56,8 @@ public void handleCommand(String channelName, Command command) {
                 }
                 return;
             }
-        } else if (CHANNEL_SIREN.equals(channelName) && command instanceof OnOffType) {
-            getSecurityCapability().ifPresent(cap -> cap.changeSirenStatus(handler.getId(),
-                    command == OnOffType.ON ? SirenStatus.SOUND : SirenStatus.NO_SOUND));
+        } else if (CHANNEL_SIREN.equals(channelName)) {
+            logger.info("Channel '{}' is read-only", command, channelName);
```
Comment (lsiepel) [inline bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/PresenceCapability.java:60]:
```suggestion
            logger.warn("Channel '{}' is read-only", command, channelName);
```

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml [hunk 2]
```diff
@@ -14,6 +14,12 @@
 			</update-channel>
 		</instruction-set>
 
+		<instruction-set targetVersion="2">
```
Comment (lsiepel) [inline bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml:17]: You also need increase the thing property to 2.

**Review by lsiepel — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml (hunk 2 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml:17]: It is done in `ModuleType.java`, remember that Netatmo binding does not use xml based thing type definition but  dynamical.

Comment (lsiepel) [reply bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/update/instructions.xml:17]: Thanks, i see now.

File: bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/PresenceCapability.java (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/PresenceCapability.java]: Corrected

**Review by lsiepel — APPROVED** [review]

**Pull request merged**